### PR TITLE
Catering for a week-in-month scenario, eg. "XXXX-08-W02"

### PR DESCRIPTION
**Pull request by HiltonGiesenow**
I've logged some previous PRs but was using the Regex directly - now I'm using the Resolver directly and seeing how it works - this is a more comprehensive PR, including a new unit Test. This deals with a
"2nd week in August" example scenario properly ("XXXX-08-W02").

### CODE REVIEW

**Comment by tellarin** [comment]
Should we just close #1925 and #1927? It seems this one is intended to superpose them.

**Comment by HiltonGiesenow** [comment]
Yup, please do

File: .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexFormat.cs [hunk 1]
```diff
@@ -173,7 +173,8 @@ private static string FormatDateRange(TimexProperty timex)
 
             if (timex.Month != null && timex.WeekOfMonth != null)
             {
-                return $"XXXX-{TimexDateHelpers.FixedFormatNumber(timex.Month, 2)}-WXX-{timex.WeekOfMonth}";
+                // return $"XXXX-{TimexDateHelpers.FixedFormatNumber(timex.Month, 2)}-WXX-{timex.WeekOfMonth}";
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexFormat.cs]: Nit: Please delete commented out code.

**Comment by HiltonGiesenow** [comment]
I can barely manage with English most days - glad you guys are working on the other langs too. Thanks for accepting PR. Another one en route now

**Comment by HiltonGiesenow** [comment]
ah, getting confused - everything got sorted in this one - thanks

**Pull request merged**